File: Haskell/beginner/1014.hs
-- Doing imports to use Text.Printf
import Text.Printf

-- Core Function
main = do
    -- Requesting values and assign into variables
    distance_travelled <- readLn :: IO Int
    total_fuel <- readLn :: IO Float

    -- Doing the calculations
    let consumption = fromIntegral distance_travelled / total_fuel -- At this time I'm ready for it!

    -- Showing results
    printf "%.3f km/l\n" consumption

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1014.c
/*Including libs*/
#include <stdio.h>

/*Main Function*/
int main(){
    
    /*Variable declaration and initialization*/
    double distance_travelled, total_fuel, average_consumption;
    
    /*Requesting inputs to user*/
    scanf("%lf", &distance_travelled);
    scanf("%lf", &total_fuel);
    
    /*Doing the calculations*/
    average_consumption = distance_travelled / total_fuel;

    /*Putting results into screen*/
    printf("%.3f km/l\n", average_consumption);
    
    /*Ending code*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/